templates/puppetagent.conf.erb

```diff
@@ -53,9 +53,12 @@
 <% if @http_proxy_port != 'UNSET' -%>
     http_proxy_port = <%= @http_proxy_port %>
 <% end -%>
-    report = true
+    report = <%= @puppet_reports %>
     graph = true
     pluginsync = true
+    interval = <%= @puppet_run_interval %>
+    splay = true
+    splay_limit = 60s
```

Comment:
`splaylimit` without the underscore. Also should default to the default value that puppet uses and be documented in the README and come with spec tests.